app/views/admin/news_articles/_news_article_type_fields.html.erb

```diff
@@ -1,16 +1,24 @@
-<%= render "govuk_publishing_components/components/select", {
-  name: "edition[news_article_type_id]",
-  id: "edition_news_article_type_id",
-  label: "News article type",
-  heading_size: "s",
-  full_width: true,
-  value: edition.news_article_type_id,
-  error_message: errors_for(edition.errors, :news_article_type_id),
-  options: ([nil]+ NewsArticleType.all).map do |type|
-    {
-      text: type&.singular_name,
-      value: type&.id,
-      selected: edition.news_article_type_id == type&.id
-    }
-  end
-} %>
+<div class="edition-form--subtype-fields" data-format-advice="<%= NewsArticleType::FORMAT_ADVICE %>">
```

Comment:
The BEM convention uses underscores for child elements (hyphens for modifiers) so this class shoufd be `edition-form__subtype-fields `.